cluster_view.php

```diff
@@ -594,7 +594,7 @@ function get_load_heatmap($hosts_up, $host_regex, $metrics, $load_scale, $tpl_da
   if (array_key_exists($user_metricname, $metrics))
     $units = $metrics[$user_metricname]['UNITS'];
 } else {
-  if (array_key_exists($user_metricname, $metrics[key($metrics)]))
+  if ((!is_null($metrics[key($metrics)])) && array_key_exists($user_metricname, $metrics[key($metrics)]))
```

Comment:
fix: PHP Fatal error:  Uncaught TypeError: array_key_exists(): Argument #2 ($array) must be of type array, null given in /usr/share/ganglia/cluster_view.php:597